packages/react/src/VideoPlayer/components/controls/Controls.tsx

```diff
@@ -0,0 +1,304 @@
+
+  /* 4. Mute                                                    */
+
+  const renderMuteButton = useMemo(
+    () => (
+      <IconControl
+        tooltip={volume > 0 ? 'Mute' : 'Unmute'}
+        onClick={() => {
+          if (volume > 0) {
+            setPreviousVolume(volume)
+            setVolume(0)
+          } else {
+            setVolume(previousVolume)
+          }
+        }}
+      >
+        {volume > 0 ? (
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            aria-hidden="true"
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
+          >
+            <path
+              fillRule="evenodd"
+              clipRule="evenodd"
+              d="M11.553 3.064C11.686 3.12273 11.799 3.21883 11.8784 3.34062C11.9577 3.46241 12 3.60464 12 3.75V20.25C12.0001 20.3954 11.9579 20.5377 11.8786 20.6596C11.7993 20.7814 11.6863 20.8776 11.5533 20.9364C11.4204 20.9952 11.2732 21.0141 11.1296 20.9908C10.9861 20.9674 10.8525 20.9029 10.745 20.805L5.46 16H2.75C2.28587 16 1.84075 15.8156 1.51256 15.4874C1.18437 15.1592 1 14.7141 1 14.25V9.75C1 8.784 1.784 8 2.75 8H5.46L10.745 3.195C10.8526 3.0974 10.9863 3.03316 11.1297 3.01008C11.2731 2.987 11.4202 3.00608 11.553 3.065V3.064Z"
+              fill="currentColor"
+            />
+            <path
+              d="M18.718 4.222C18.8586 4.08155 19.0492 4.00266 19.248 4.00266C19.4467 4.00266 19.6374 4.08155 19.778 4.222C24.074 8.518 24.074 15.482 19.778 19.778C19.6358 19.9105 19.4478 19.9826 19.2535 19.9792C19.0592 19.9757 18.8738 19.897 18.7364 19.7596C18.599 19.6222 18.5203 19.4368 18.5168 19.2425C18.5134 19.0482 18.5855 18.8602 18.718 18.718C19.6003 17.8358 20.3002 16.7885 20.7777 15.6358C21.2552 14.4831 21.501 13.2477 21.501 12C21.501 10.7523 21.2552 9.51687 20.7777 8.36419C20.3002 7.21151 19.6003 6.16417 18.718 5.282C18.5775 5.14137 18.4987 4.95075 18.4987 4.752C18.4987 4.55325 18.5775 4.36263 18.718 4.222V4.222Z"
+              fill="currentColor"
+            />
+            <path
+              d="M16.243 7.757C16.1733 7.68733 16.0906 7.63207 15.9996 7.59437C15.9086 7.55667 15.811 7.53726 15.7125 7.53726C15.614 7.53726 15.5164 7.55667 15.4254 7.59437C15.3344 7.63207 15.2517 7.68733 15.182 7.757C15.1123 7.82667 15.0571 7.90937 15.0194 8.0004C14.9817 8.09142 14.9623 8.18898 14.9623 8.2875C14.9623 8.38602 14.9817 8.48358 15.0194 8.5746C15.0571 8.66563 15.1123 8.74833 15.182 8.818C15.5999 9.23586 15.9313 9.73194 16.1575 10.2779C16.3836 10.8239 16.5 11.409 16.5 12C16.5 12.591 16.3836 13.1761 16.1575 13.7221C15.9313 14.2681 15.5999 14.7641 15.182 15.182C15.0495 15.3242 14.9774 15.5122 14.9808 15.7065C14.9843 15.9008 15.063 16.0862 15.2004 16.2236C15.3378 16.361 15.5232 16.4397 15.7175 16.4432C15.9118 16.4466 16.0998 16.3745 16.242 16.242C17.3671 15.1168 17.9992 13.5907 17.9992 11.9995C17.9992 10.4083 17.3671 8.88221 16.242 7.757H16.243Z"
+              fill="currentColor"
+            />
+          </svg>
```

Comment:
Can we replace this with an Octicon please? Looks like we have one [here](https://primer.style/design/foundations/icons/unmute-24).

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/3476819281a2ff27b50715bf57261befc6f1d0f5